### PR TITLE
Added Flutter for contributions

### PR DESCRIPTION
**Pull request by mohak852**
Added Flutter for contribution in the First contribution.
Screenshot from 2020-05-30 01-57-39

### CODE REVIEW

File: src/components/ProjectList/listOfProjects.js [hunk 1]
```diff
@@ -212,6 +212,13 @@ const projectList = [
     description: 'React Styleguidist is a component development environment with hot reloaded dev server and a living style guide that you can share with your team. It lists component propTypes and shows live, editable usage examples based on Markdown files.',
     tags: ['JavaScript', 'UI'],
   },
+  {
+    name: 'Flutter',
+    imageSrc: 'https://raw.githubusercontent.com/flutter/website/master/src/_assets/image/flutter-lockup.png',
+    projectLink: 'https://github.com/flutter/flutter/contribute',
+    description: 'Flutter makes it easy and fast to build beautiful apps for mobile and beyond.',
+    tags: ['Dart','Android','Flutter','UI'],
```
Comment (TomerPacific) [inline src/components/ProjectList/listOfProjects.js:220]: Flutter is also for Web and iOS.

Comment (sara-02) [reply src/components/ProjectList/listOfProjects.js:220]: Also, include the tag `Cross-platform` as it can be used for web/desktop apps as well.

Comment (mohak852) [reply src/components/ProjectList/listOfProjects.js:220]: Screenshot from 2020-05-31 01-05-08

**Comment by mohak852** [comment]
You can review that I have added a "cross-platform" tag in flutter

**Pull request merged**